src/style.css

```diff
@@ -1,3 +1,67 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
 body {
   margin: 0;
+  --blue-color: #00acdc;
```

Comment:
It is customary to place variables separately in the `:root` element.
Also, the name of the variable should show its purpose, not the content, because in the development process the color can change and you will have to change it everywhere, not only in the variable definition.